src/utils/install/journal_defaults.json

```diff
@@ -3064,16 +3064,31 @@
             "name": "general"
         },
         "setting": {
-            "description": "Enable if you want to hide review metadata from authors (date assigned, decision, date completed).",
+            "description": "Enable if you want to show review metadata to authors (date assigned, decision, date completed).",
             "is_translatable": false,
-            "name": "hide_review_metadata_from_authors",
-            "pretty_name": "Hide Review Metadata from Authors",
+            "name": "enable_peer_review_data_on_review_page",
+            "pretty_name": "Enable Peer Review Data on Review Pages",
             "type": "boolean"
         },
         "value": {
             "default": ""
           }
     },
```

Comment:
This needs a data migration to delete the old setting in people's databases, right?